apps/sms/views/conversation/style/compose_view.css

```diff
@@ -0,0 +1,111 @@
+.new .article-list {
```

Comment:
hmm, let's raise this question to bugzilla because I have no better idea about the naming, but it would be great if we can have a clear naming about the compose widget and compose view.